### DISCUSSION TITLE
container files in the file repository

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1231] [users: ideas for new features]
Moved here from "ideas and drafts for the recording system" in the wiki:

To upload files that belong together and which come in a huge number, we could allow for container files. Could be a ZIP file with some conventions, e.G. it could containt some python for installing the packet or it could contain a preview, so that it can be displayed in the file repository.
Here we also have files that are not allowed to be send towards filemaker. Some of the files might be normal files, bound to the packet, but managed by the file repository.
We could think of a payload that comes with a file. The file is managed by the file repos in a normal way, the paypload is connected to it and in the background. When downloading such a container, all of it (or sometimes only parts of it?) is downloaded.

**Comment by urapadmin** [comment]
see #43 which is about uploading zips

**Comment by urapadmin** [comment]
#138 is about different vresions of a file, which is also connected to this here in a way.

**Comment by lbestock** [comment]
Not sure if related but this one reminds me of our need to figure out how to deal with 3D models in the file repository. They are a final file (huge) built with references to a bunch of images, and the relations have to be maintained for the thing to work; the filepath thing has always tripped me up.

**Comment by lbestock** [comment]
Alex has asked me for all the old GIS stuff so he can start working on standardizing it and it made me think a) that I would get it to him this way if this existed and b) the GIS data really belongs archived with all our other data, too.
